app/assets/javascripts/orders.coffee

```diff
@@ -144,14 +145,13 @@ app.controller 'CheckoutController', ($scope, Product, Order, Address, CreditCar
     $scope.order.state = 'placed'
     $scope.order.$save()
     .then ->
-      console.log "success"
+      delete $scope.orderErrors
     .catch (result) ->
-      console.log "error"
+      $scope.orderErrors = result.data.orders
```

Comment:
I think it's tied to the controller ( i.e. credit_cards_controller.rb, address_controller.rb, etc... ). I'll change the naming to be more consistent.